docs/how-to/refresh/refresh-multi-cluster.md

```diff
@@ -0,0 +1,38 @@
+# How to refresh a multi-cluster deployment
+
+A MySQL multi-cluster deployment (also known as a multi-node cluster or cluster set) can be upgraded by performing a refresh of each cluster individually.
+
+This guide goes over the steps and important considerations before refreshing multiple MySQL clusters.
+
+## Determine cluster order
+
+To upgrade a multi-cluster deployment, each cluster must be refreshed one by one - starting with the standby clusters.
+
+**The primary cluster must be the last one to get refreshed.**
+
+When a primary cluster gets refreshed, it triggers a potentially costly re-election process. To minimise this cost, all standby clusters should be refreshed before the primary.
+
+<!--TODO: Mention how to identify primary cluster-->
+
+## Refresh each cluster
+
+For each cluster, follow the instructions in [](/how-to/refresh/refresh-single-cluster).
+
+**Perform a health check before proceeding to the next cluster.**
+
+Use the [`get-cluster-status`](https://charmhub.io/mysql/actions#get-cluster-status) Juju action to check that everything is healthy after refreshing a cluster.
+
+<!---TODO: example of running get-cluster-status (and making sure the cluster-set param is True?)
```

Comment:
For each cluster, it makes more sense to run with `cluster-set=false`  (or no parameter, as this is default), since we want to evaluate the updated cluster health, i.e. if all units are online